h/static/scripts/annotator/anchoring/pdf.coffee

```diff
@@ -249,6 +251,10 @@ exports.describe = (root, range, options = {}) ->
   startTextLayer = getNodeTextLayer(range.start)
   endTextLayer = getNodeTextLayer(range.end)
 
+  if startTextLayer is null
```

Comment:
If this is unintentional behavior in annotator then I think it seems to come down to [this line](https://github.com/hypothesis/client/blob/9e5755dad3c02fc0f196b6e776aefd2f0f9174d5/h/static/scripts/vendor/annotator.js#L628) in annotator. It looks as if that code might not have considered that [nextSibling](https://developer.mozilla.org/en-US/docs/Web/API/Node/nextSibling) can return `null`. Here's [the commit that introduced the line](https://github.com/openannotation/annotator/commit/494d01f4131996d6b8bc17367b08245d6febf97a), you can see that the intention of the line was to "avoid splitting off zero-length pieces" and to "be more conservative when slicing and dicing elements".

I'm not familiar with annotator and I'm not sure I entirely understand what that code is meant to be doing.

We could make a change annotator along the lines of `nr.start = r.start.nextSibling || r.start;`.

The relevant code seems to be completely gone from newer versions of annotator, though.